aio/content/examples/toh-pt6/src/app/hero-detail/hero-detail.component.ts

```diff
@@ -24,7 +24,7 @@ export class HeroDetailComponent implements OnInit {
   }
 
   getHero(): void {
-    const id = +this.route.snapshot.paramMap.get('id');
+    const id = parseInt(this.route.snapshot.paramMap.get('id') as string, 10);
```

Comment:
Earlier I think you did something different, like:

```suggestion
    const id = parseInt(this.route.snapshot.paramMap.get('id') || '0', 10);
```